apps/client/src/autofill.js

```diff
@@ -0,0 +1,30 @@
+import { getRestrictionByTypeName } from "./utils";
+
+export const autofillResolvers = {
+  monumentBoolean: ({ address }) =>
+    address?.restrictions
+      ? !!getRestrictionByTypeName(address.restrictions, "Monument")
+      : undefined,
```

Comment:
Can this not be `address?.restrictions && !!getRestrictionByTypeName(address.restrictions, "Monument")` or do you need to specifically return `undefined`?

Reply:
Good catch, will fix.